sp800-63-3/sec3_definitions.md

```diff
@@ -138,13 +135,13 @@ As defined by OMB Circular [[A-130]](#A-130), Personally Identifiable Informatio
 The secret part of an asymmetric key pair that is used to digitally sign or decrypt data.
 
 #### Pseudonymous Identifier
-A meaningless, but unique number that does not allow the RP to infer the subscriber but which does permit the RP to associate multiple interactions with the subscriber’s claimed identity.
+A meaningless but unique number that does not allow the RP to infer the subscriber but which does permit the RP to associate multiple interactions with the subscriber’s claimed identity.
```

Comment:
Maybe "infer anything regarding the subscriber" rather than "infer the subscriber"